HW04.Rmd

```diff
@@ -29,35 +48,76 @@ Knitr is part of the knitr package. Allows you to change column names and add ca
 
 
 ```{r month, warning=FALSE}
+gun_deaths_month <- gun_deaths %>%
+    group_by(month) %>%
+    count(month) %>%
+    rename(number = n)
 
+kable(gun_deaths_month, format = "markdown")
 ```
 
 
 ### Generating a bar chart with human-readable labels on the x-axis. That is, each month should be labeled "Jan", "Feb", "Mar" (full or abbreviated month names are fine), not `1`, `2`, `3`.
 
 ```{r month_chart}
-
+# create human-readable label for each month
+gun_deaths$month <- factor(gun_deaths$month, labels = c("Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"))
+

```

Comment:
I think you can try the built-in variable "month.abb" so that you do not have to type in all the months by yourself.

Reply:
Thank you for the great advice!